File: SoloLearn_CodeCoach/Easy/JungleCamping/JungleCamping.cpp
#include <iostream>
#include <string>
#include <map>

int main() {
    std::map<std::string, std::string> noiseToAnimal = {
        {"Grr", "Lion"}, {"Rawr", "Tiger"},
        {"Ssss", "Snake"}, {"Chirp", "Bird"}
    };

    std::string input;
    while (std::cin >> input) {
        auto it = noiseToAnimal.find(input);
        std::string output = it != noiseToAnimal.end() ? it->second : "";
        std::cout << output << ' ';
    }
    return 0;
}

File: SoloLearn_CodeCoach/Easy/JungleCamping/JungleCamping.java
import java.util.Map;
import java.util.Scanner;

public class JungleCamping {
    public static void main(String[] args) {
        Map<String, String> noiseToAnimal = Map.of(
                "Grr", "Lion",
                "Rawr", "Tiger",
                "Ssss", "Snake",
                "Chirp", "Bird");

        Scanner input = new Scanner(System.in);
        String[] noises = input.nextLine().split(" ");

        String animals = "";

        for (String noise: noises) {
            animals += noiseToAnimal.get(noise) + " ";
        }

        System.out.println(animals);
    }
}
